src/test/java/com/scriptbasic/TestEngine.java

```diff
@@ -335,6 +335,18 @@ public void testEvalStringSW() throws Exception {
             // END SNIPPET: helloWorldStringSW
         }
 
+        @Test
+        @DisplayName("multiple statement on one line")
+        public void testEvalStringMP() throws Exception {
+            // START SNIPPET: helloWorldStringSW
```

Comment:
You should not reuse the same snippet identifier in a project. It will kill the document generation. Do not mark a code fragment to be a snippet only if you use that in the documentation.